practical_09/taxi_simulator.py

```diff
@@ -0,0 +1,96 @@
+"""
+Taxi simulator to allow customer to drive a taxi of their choosing
+"""
+
+from practical_09.silver_service_taxi import SilverServiceTaxi
+from practical_09.taxi import Taxi
+
+MENU = """
+q)uit, c)hoose, d)rive
+>>> """
+
+taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
```

Comment:
Remember uppercase for constants. The current name suggests a (mutable) global variable, which is not recommended.